README.md

```diff
@@ -14,6 +14,7 @@ The script relies on certain developer tools to be available when running.
 
 - git - To initialise the project as a repository.
 - composer - To download the dependencies. If composer is not present, the dependencies don't get downloaded.
+- lando - Docker driven local development tool.
```

Comment:
This is not accurate. We don't need `lando` when running axl-template. In fact, `lando` is not even *required* for running the site generated by axl-template. It only generates Lando specific files if you run `init-drupal` with `--lando` option or you directly run `init-lando`.

I would suggest add a new section in existing "Requirements" section to cover these optional dependencies.